src/js/components-react/Tabs.js

```diff
@@ -24,15 +24,15 @@ const TabsReact = props => {
         disabled={item.isDisabled}
         disabledClassName="kuc-tabs-disabled"
       >
-        {item.label}
+        {item.tabName}
       </Tab>
     );
   });
 
   const tabPanel = props.items.map((item, i) => {
     return (
       <TabPanel key={i}>
-        <span dangerouslySetInnerHTML={{__html: item.value}} />
+        <span dangerouslySetInnerHTML={{__html: item.tabContent}} />
```

Comment:
dangerouslySetInnerHTML shouldn't be used. It should be something like this:
<span>
{item.tabContent}
</span>

Reply:
Fixed in latest commit